templates/ssl_ratios_raw_v1.sql

```diff
@@ -0,0 +1,24 @@
+SELECT
+  submission_date,
+  os,
+  country,
+  SUM(udf.get_key(histogram_parent_http_pageload_is_ssl, 0)) AS non_ssl_loads,
+  SUM(udf.get_key(histogram_parent_http_pageload_is_ssl, 1)) AS ssl_loads,
+  -- ratio of pings that have the probe
+  COUNT(histogram_parent_http_pageload_is_ssl) / COUNT(*) AS reporting_ratio
+FROM
+  telemetry.main_summary_v4
+WHERE
+  sample_id = 42
+  AND normalized_channel = 'release'
+  AND os IN ('Windows_NT', 'Darwin', 'Linux')
+  AND app_name = 'Firefox'
+  AND submission_date > DATE '2016-11-01'
+  AND (submission_date = @submission_date
+    OR @submission_date IS NULL)
+GROUP BY
+  submission_date,
+  os,
+  country
+HAVING
+  non_ssl_loads + ssl_loads > 5000
```

Comment:
putting this here instead of the end of the view causes groups under this threshold to be left out of calculating `total_loads`, but it reduces data stored and it might be better? needs review

Reply:
This seems fine, although I have no context of analyzing this data to know whether 5000 is a reasonable threshold here.

Reply:
ah, my bad. @jklukas I flagged you hoping for review on the rest of the PR from someone who hadn't written it, and I no longer meet that qualification.

this bit needs review from someone who actually uses the data, or knows how it's being used. @mreid-moz could you review this decision or tag someone appropriate?

Reply:
How much impact does this have on the amount of data stored? I favour exactly matching the existing query, then refactoring things once we have a replacement for the data we're publishing now.

That said, if @chutten has thoughts, I'd love to hear them!

Reply:
okay, i'll move this to exact match

Reply:
It was an arbitrary limit to ensure that in the final result set we weren't publishing rows derived from fewer than 5k pageloads. There was no formal review, so we might not have needed the limit at all given the low number of dimensions